pkg/pillar/types/contenttreetypes.go

```diff
@@ -114,7 +114,6 @@ type ContentTreeStatus struct {
 	HasResolverRef    bool
 	HasDownloaderRef  bool
```

Comment:
Can we get rid of `HasDownloaderRef` and `HasVerifierRef` too, since all of that is on `BlobStatus` now?

Reply:
Good catch. Done